creation/views.py

```diff
@@ -12,11 +12,11 @@ def create(request):
 
         if form.is_valid():
             event = form.save(commit=False)
-            print(request)
             event.created_date = timezone.now()
             event.access_code = genAccessCode()
-
             event.save()
+            p = Group(event=event)
```

Comment:
We should be creating a Group here we will be creating them later when we start letting user join the event which is what we will be working next.